### PR TITLE
Added command to retrieve following users

### LINKED ISSUE #4: Add command to list all following
This command should fetch all the following users of a user.

Suggested command:

gac list following <username>

Output: Could be a list or table of content.

Reference:

[API for the fetching following](https://api.github.com/users/thamaraiselvam/following)

### PR DESCRIPTION
**Pull request by raf924**
Closes #4

### CODE REVIEW

**Comment by thamaraiselvam** [comment]
can you write unit test cases? it dropped from 28% to 17%.

**Comment by raf924** [comment]
@thamaraiselvam there you go

File: cmd/types/types.go [hunk 1]
```diff
@@ -6,3 +6,12 @@ type UserInfo struct {
...
 }
+
+//FollowingUser contains following user information
+type FollowingUser struct {
+	Name string `json:"login"`
+	URL  string `json:"html_url"`
+}
+
+//FollowingUserList stores a list of FollowingUser
+type FollowingUserList = []FollowingUser
```
Comment (thamaraiselvam) [inline cmd/types/types.go]: Renamed `FollowingUserList ` to  `FollowingUsers`

File: cmd/following.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	client := service.CreateClient(fmt.Sprintf("/users/%s/following", name))
+	userInfoList, err := client.GetFollowing()
+	if err != nil {
+		_ = fmt.Errorf("%v", err)
+		os.Exit(1)
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 5, ' ', 0)
+	_, _ = fmt.Fprintln(w, "Name\tGithub Link")
+	for _, userInfo := range userInfoList {
```
Comment (thamaraiselvam) [inline cmd/following.go]: For displaying data add a separate method so it can be testable.

**Review by thamaraiselvam — COMMENTED** [review]
Please fix merge conflict and resolve comments

File: cmd/following.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package cmd
```
Comment (thamaraiselvam) [inline cmd/following.go]: Move this command under list command. Check followers command for reference

**Review by thamaraiselvam — CHANGES_REQUESTED** [review]
I really appreciate your PR @raf924 

Please fix the review comments.

File: cmd/types/types.go (hunk 1 above)
Comment (raf924) [reply cmd/types/types.go]: Done

File: cmd/following.go (hunk 3 above)
Comment (raf924) [reply cmd/following.go]: Done

File: cmd/following.go (hunk 2 above)
Comment (raf924) [reply cmd/following.go]: Done

**Comment by raf924** [comment]
@thamaraiselvam will you merge now ?

**Review by thamaraiselvam — APPROVED** [review]

**Pull request merged**